src/style.css

```diff
@@ -1 +1,54 @@
 /* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('images/star.svg');
+  background-size: auto;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
```

Comment:
Avoid fixing container size if there is no such requirement. Let the content size dictate it. To avoid overflow or accidental scroll bar, it would be better not to set a fixed width and height for `.stars__star`.